azurerm/internal/services/network/registration.go

```diff
@@ -102,6 +102,7 @@ func (r Registration) SupportedResources() map[string]*schema.Resource {
 		"azurerm_route_filter":                                                           resourceArmRouteFilter(),
 		"azurerm_route_table":                                                            resourceArmRouteTable(),
 		"azurerm_route":                                                                  resourceArmRoute(),
+		"azurerm_security_partner_provider":                                              resourceArmSecurityPartnerProvider(),
```

Comment:
I think we should have this be a little more specific? what is this security partner applied to? vhubs? then should this be:
```suggestion
		"azurerm_virtual_hub_security_partner_provider":                                              resourceArmSecurityPartnerProvider(),
```

Reply:
updated